docs/src/main/asciidoc/secret-management.adoc

```diff
@@ -56,23 +57,25 @@ spring:
           endpoint: ${AZURE_KEYVAULT_ENDPOINT}
 ----
 
+If your application is authenticated by other methods like Managed Identity or Azure CLI, properties like `tenant-id`, `client-id`, `client-secret` is not necessary. But if these properties are configured, then these properties have higher priority. Please refer to link:authentication.html[Authentication section] to get more information.
+
 ===== Java code
 
 [source,java]
 ----
 @SpringBootApplication
 public class KeyVaultSample implements CommandLineRunner {
 
```

Comment:
Have you tested it?

Reply:
No, it's just code snippet, not project, no need to test, I think.